NAMESPACE

```diff
@@ -1,7 +1,9 @@
 # Generated by roxygen2: do not edit by hand
 
+export(add_ws_to_parse_data)
```

Comment:
I usually use the combination 
`#' @export`
`#' @keywords internal `
and if I want to call the function during the development phase, I can save some keystrokes since I write `hidden_function()` instead of `package:::hidden_function()`. But I assume that's not best practice. Do you prefer to export only top-level functions right from the beginning then I guess?